### ISSUE TITLE
Missing Citeproc. Can you update pandoc?

### ISSUE THREAD

**Issue by devantler** [thomasWeise/docker-texlive#5]
I am working on <https://github.com/niem94/template.pandoc-markdown>. It is a repository template for writing academic documents with markdown utilizing pandoc to compile to pdf. The only thing missing is a working GitHub Action. I cant use the official pandoc-latex image, as I have advanced needs. And yours is almost working for my use-case. Only problem is that the citeproc option for pandoc is not supported:

<img width="393" alt="image" src="https://user-images.githubusercontent.com/26203420/99923771-8ef7fc80-2d37-11eb-95e1-fc132def6f96.png">

I believe citeproc is included as a part of pandoc today, and I really need a docker image with a full texlive distribution and pandoc with citeproc. I would be so happy if you can update pandoc to the newest version :-)

**Comment by thomasWeise** [comment]
Hi.

Please take a look at my other repository https://github.com/thomasWeise/docker-pandoc/ providing the image `thomasweise/docker-pandoc`.
That one does include a pandoc installation with citeproc based on a (thin) texlive installation.
You can use citeproc with `--filter pandoc-citeproc`.

Cheers,
   Thomas.

**Comment by devantler** [comment]
Thank you, I was not aware, that I had to add the filter, but it makes sense! :-) 

Now that I have your attention, I actually have two feature requests. A filter to convert markdown plantuml code blocks into an their respective image. And a filter to be able to use glossaries in markdown. 

I know these two works wonders, as I use them locally, and if you feel they make sense in your docker image, I would be a happy man!

https://github.com/tomncooper/pandoc-gls
https://github.com/timofurrer/pandoc-plantuml-filter

**Comment by thomasWeise** [comment]
Hi.

Sorry, I am current busy with other things.

I am using this pandoc container mainly for my personal book builder tool chain.
Thus, I only want to have things in it that a) do not make the image much bigger and b) work with arbitrary pandoc output formats.
All the current plugins and components work with pdf, html, and epub.

pandoc-gls seems to be mainly for tex-based conversations and maybe does not work with HTML or ebook targets(?)
If that is the case, then I have no use for it in my personal tool chain.

pandoc-plantuml-filter will probably require me to get python pip to work which probably does not work in the current version (I suspect).
I do not want to mess with that as it would probably require quite some testing work and may make my image significantly bigger (or not, maybe it would work straightforward, not sure).

The bottom line is that I currently do not need these tools and do not have the time to invest to update my container to make sure they work properly.

Please feel free to make an image based on mine installing these features. You can either directly base it on my container or fork my repo to make a new one.

All the best,
   Thomas.

**Comment by devantler** [comment]
That is very fair :-) I will look into making my own docker image, with the tools I need. Thank you for your time Thomas, and for your permission to base my image on yours in case that is the direction I will go. 

Best of luck,
Nikolai
